Set flags for libraries that link with Xft to meet normal Solaris rules

--- xft.pc.in	Wed Apr 30 17:49:20 2003
+++ xft.pc.in	Wed Aug 24 18:59:14 2005
@@ -11,5 +11,5 @@
 Description: X FreeType library
 Version: @VERSION@
 Requires: fontconfig
-Libs: -L${libdir} -lXft -lX11 ${freetypelibs} ${xrenderlibs}
+Libs: -L${libdir} -R${libdir} -lXft
 Cflags: -I${includedir} ${freetypecflags} ${xrendercflags}
